db/migrate/20210908204646_add_loan_prefix_to_response_set_kinds.rb

```diff
@@ -0,0 +1,9 @@
+class AddLoanPrefixToResponseSetKinds < ActiveRecord::Migration[6.1]
+  def up
+    execute("UPDATE response_sets SET kind = 'loan_' || kind")
```

Comment:
Goal here is to get rid of all the places in the code where we are either adding on this prefix or stripping it 🤮